### DISCUSSION TITLE
enemy db

### DISCUSSION THREAD

**Discussion by k0l11** [genshinsim/gcsim#823] [Ideas / Features Request]
This is useful for hp mode.
Let's say I want to sim against a single Lvl. 100 ruin guard. With the way sim is at the moment, I'd have to go lookup the HP, particle breakpoints and amount/type of particles dropped. If we had an enemy db, then this would be as simple as writing `target lvl=100 name=ruinguard;` (syntax is just an example)

**Comment by srliao** [comment]
hmm could probably start this by supporting a couple common targets first

actually i wonder if this info can be pulled directly out of datamine

**Comment by k0l11** [comment]
What I forgot to mention is that we also need some sort of param for the special hp multiplier that exists. Abyss is usually a x2.5 multiplier but in rare cases they deviate from that, like in one abyss the Golden Wolflord had a x2.0 multiplier instead of x2.5.
https://genshin-impact.fandom.com/wiki/Level_Scaling/Enemy#Special_Multiplier

**Comment by k0l11** [comment]
done in v2.22.0
